front/app/utils/containers/clickOutside/index.tsx

```diff
@@ -1,12 +1,14 @@
-import React, { PureComponent, MouseEvent } from 'react';
```

Comment:
Used AI here for initial conversion, and then did a manually check. Made a couple tweaks, but looked good to me otherwise.